packages/govuk-frontend/src/govuk/helpers/_typography.scss

```
@@ -156,17 +156,37 @@
 /// @access public
 
 @mixin govuk-font-size($size, $line-height: false, $important: false) {
-  @if not map-has-key($govuk-typography-scale, $size) {
-    @error "Unknown font size `#{$size}` - expected a point from the type scale.";
+  $internal-use: str-slice(#{$size}, 1, 1) == "_";
```

Could we add a comment just to clarify what this is? You and me both know obviously, but for future folks.

Aside: Should we keep this in as a means to deprecate other parts of the type scale in future or remove it in 6.0?

---

Yeah course, will do. Thanks @owenatgov

Keep it? We've built ourselves a mechanism to deprecate sizes that other teams could use too

---

Aha good point. Let's leave it in.

---

Any thoughts on a better variable than `$internal-use`?

I thought about `$size-is-deprecated` but that's only when the map _also_ has a `deprecated` property

---

I've gone with `$size-internal-use-only` and will push up this comment:

```suggestion
  // Flag font sizes that start with underscores so we can suppress warnings on
  // deprecated sizes used internally, for example `govuk-font($size: "_14")`
  $size-internal-use-only: str-slice(#{$size}, 1, 1) == "_";
```

---

I think that's fine. This particular variable feels really tricky to name without getting trite and what you've gone with is probably as good as we can get it.